README.md

```diff
@@ -49,8 +74,17 @@ pip3 install -r requirements.txt
 ## Getting started
 Clarifai uses **Personal Access Tokens(PATs)** to validate requests. You can create and manage PATs under your Clarifai account security settings.
 
+[Create PAT:](https://docs.clarifai.com/clarifai-basics/authentication/personal-access-tokens/)
+* ***Log into Portal &rarr; Profile Icon &rarr; Security Settings &rarr; Create Personal Access Token &rarr; Set the scopes &rarr; Confirm***
+
+
 Export your PAT as an environment variable. Then, import and initialize the API Client.
 
+[Get User ID:](https://help.clarifai.com/hc/en-us/articles/4408131912727-How-do-I-find-my-user-id-app-id-and-PAT-)
+
+* ***Log into Portal &rarr; Profile Icon &rarr; Account &rarr; Profile &rarr; User-ID***
+
+Set PAT as environment variable through terminal:
```

Comment:
Maybe make "Create PAT" as a bullet point, and put the description on the same line? 
Same for User ID. 
Then, group both bullets together above the line "Export your PAT...".

Reply:
![image](https://github.com/Clarifai/clarifai-python/assets/147609006/d3510181-b9f0-4e4d-9fe8-99343ef702fb)

Is this fine?

Reply:
Yes, thanks!